install.ps1

```diff
@@ -468,10 +468,12 @@ white_list:
   proxyPorts:
     - 9796
 "@
-        Set-Content -Path $env:CATTLE_AGENT_CONFIG_DIR/config -Value $winsConfig
```

Comment:
didnt we have this on purpose so it resets the contents of the config on a reboot so we dont append forever?

Reply:
@luthermonson why would this run on a reboot? This just run once during registration and the other stuff does an append also.

Reply:
we have the upgrade process that will be calling this i guess is what i was referring too

Reply:
https://github.com/rancher/system-agent/blob/main/package/suc/run.sh#L48

Reply:
There was an (undocumented) change in behavior as we only use to append to the wins config file and now we replace any existing wins config file. I am setting the behavior back to appending only as it makes it impossible to test custom wins configuration and may impact our community end users. In my case, I was trying to test my changes while writing this PR by manually creating a wins config with `tls-config` and it would be wiped out when the install script was run.